docs/source/logging/index.md

```diff
@@ -1,19 +1,182 @@
-2. Your project-side logging configuration. Every project generated using Kedro's CLI `kedro new` command includes a file `conf/base/logging.yml`. You can alter this configuration and provide different configurations for different run environment according to the [standard Kedro mechanism for handling configuration](../configuration/configuration_basics.md).
+Kedro uses [Python's `logging` library](https://docs.python.org/3/library/logging.html). Configuration is provided as a dictionary according to the [Python logging configuration schema](https://docs.python.org/3/library/logging.config.html#logging-config-dictschema) in Kedro's default logging configuration, as described below.
+
+By default, Python only shows logging messages at level `WARNING` and above. Kedro's logging configuration specifies that `INFO` level messages from Kedro should also be emitted. This makes it easier to track the progress of your pipeline when you perform a `kedro run`.
+
+# Default logging configuration
+Kedro's [default logging configuration](https://github.com/kedro-org/kedro/blob/main/kedro/framework/project/default_logging.yml) defines a handler called `rich` that uses the [Rich logging handler](https://rich.readthedocs.io) to format messages. We also use the Rich traceback handler to render exceptions.
+
+## How to perform logging in your Kedro project
+To add logging to your own code (e.g. in a node):
+
+```python
+import logging
+
+logger = logging.getLogger(__name__)
+logger.warning("Issue warning")
+logger.info("Send information")
+logger.debug("Useful information for debugging")
+```
+
+You can use Rich's [console markup](https://rich.readthedocs.io/en/stable/markup.html) in your logging calls:
+
+```python
+log.error("[bold red blink]Important error message![/]", extra={"markup": True})
+```
+
+## How to customise Kedro logging
+
+To customise logging in your Kedro project, you need to specify the path to a project-specific logging configuration file. Change the environment variable `KEDRO_LOGGING_CONFIG` to override the default logging configuration. Point the variable instead to your project-specific configuration, which we recommend you store inside the project's`conf` folder, and name `logging.yml`.
+
+For example, you can set `KEDRO_LOGGING_CONFIG` by typing the following into your terminal:
+
+```bash
+export KEDRO_LOGGING_CONFIG=<project_root>/conf/logging.yml
+```
+
+After setting the environment variable, any subsequent Kedro commands use the logging configuration file at the specified path.
 
 ```{note}
-Providing project-side logging configuration is entirely optional. You can delete the `conf/base/logging.yml` file and Kedro will run using the framework's built in configuration.
+If the `KEDRO_LOGGING_CONFIG` environment variable is not set, Kedro will default to using the logging configuration file at the project's default location of Kedro's `default_logging.yml`.
```

Comment:
This doesn't quite make sense but I don't know how to fix it @noklam 

I think maybe it's

```suggestion
If the `KEDRO_LOGGING_CONFIG` environment variable is not set, Kedro uses the logging configuration file at the project's default location, which is named`default_logging.yml`.
```

Where is this default location?